### PR TITLE
k8s runner for mlbox

### PR DESCRIPTION
**Pull request by swiftdiaries**
[Proposal Reference](http://bit.ly/box-k8s-runner)

- [x] k8s runner cli
- [x] CI for k8s runner
- [x] PyPI releaser action in CI
- [x] tests for cli functions
- [x] docs for k8s runner

### CODE REVIEW

**Comment by sergey-serebryakov** [comment]
Looks good to me! I have not ran the runner, just looked at the code. Several comments:
- Please, change the version from 0.2.1 to 0.2.2 (setup.py and requirements.txt). I think it should be done after syncing with master.
- This is just a suggestion. I think there's a duplicate code in file k8s_run.py (lines 35-48 and 51-64). Can that block be implemented as a method?

**Comment by sub-mod** [comment]
kindly rebase and remove the merge commit.

**Comment by swiftdiaries** [comment]
Thanks for the comments ! 
I'll commit changes before EOD today

File: docs/runners/kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+schema_type: mlbox_invoke
+schema_version: 1.0.0
+
+task_name: kubernetes   # task name set to 'kubernetes'
+
+input_binding:  # input parameters (name: value)
+  data_dir:
+    path: workspace/data
+    k8s:
+      persistentVolumeClaim: mlbox-input
```
Comment (xyhuang) [inline docs/runners/kubernetes.md:37]: how about using snake_case here to make it consistent with the rest of the config file? e.g. "persistent_volume_claim" or just "pvc"

Comment (swiftdiaries) [reply docs/runners/kubernetes.md]: changed it to pvc

**Comment by swiftdiaries** [comment]
Rebased with remote master and moved duplicated code into a function

**Comment by xyhuang** [comment]
LGTM

**Comment by swiftdiaries** [comment]
Thanks for the reviews and merge :)

**Pull request merged**